.github/workflows/PROJECT-PR-COMMENT.yaml

```
@@ -0,0 +1,57 @@
+        id: check-response
+        run: |
+          if [ "${{ steps.api-call.outputs.status }}" == "200" ]; then
+            echo "API call successful"
+            echo "COMMENT_BODY=$(echo '${{ steps.api-call.outputs.response }}' | jq -r '.commentMarkdown')" >> $GITHUB_ENV
+          else
+            echo "API call failed with status ${{ steps.api-call.outputs.status }}"
+            echo "COMMENT_BODY=API 호출에 실패했습니다. 레포지토리가 허용 목록에 있는지 확인하세요." >> $GITHUB_ENV
+          fi
+      
+      - name: Add comment to PR
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)